src/ValueParsers/TimeParser.php

```diff
@@ -7,12 +7,13 @@
 use InvalidArgumentException;
 
 /**
- * ValueParser that parses the string representation of a time.
+ * ValueParser that parses different ISO string representations of time values.
```

Comment:
"various", not "different". Maybe also "formats resembling ISO 8601" or "compatible with ISO 8601", to indicate that our interpretation doesn't follow ISO exactly.